src/js/fetchBooks.js

```diff
@@ -0,0 +1,13 @@
+import Notiflix from 'notiflix';
+
+export async function fetchBooks() {
```

Comment:
we use axios library in this project to collect API's from back-end. Please restructure the code for axios